src/app/modules/item/components/item-edit-wrapper/item-edit-wrapper.component.ts

```diff
@@ -139,7 +139,7 @@ export class ItemEditWrapperComponent implements OnInit, OnChanges, OnDestroy, P
     const usesApi = formControls.usesApi?.value as boolean;
     if (isNotUndefined(this.initialFormData.usesApi) && usesApi !== this.initialFormData.usesApi) itemFormValues.uses_api = usesApi;
 
-    const textIdValue = (formControls.textId?.value as string).trim();
+    const textIdValue = ((formControls.textId?.value as string | null) || '').trim();
```

Comment:
That's was the problem, because of forced type to string I've missed that it could be `null` value which not accepted `.trim()`. As fix I've add force convert to string like: `null || ''`

Reply:
ah yes, sorry